core/config/project_settings.cpp

```diff
@@ -1259,6 +1259,19 @@ ProjectSettings::ProjectSettings() {
 	CRASH_COND_MSG(singleton != nullptr, "Instantiating a new ProjectSettings singleton is not supported.");
 	singleton = this;
 
+#ifdef TOOLS_ENABLED
+	// Available only at runtime in editor builds. Needs to be processed before anything else to work properly.
+	if (!Engine::get_singleton()->is_editor_hint()) {
+		String editor_features = OS::get_singleton()->get_environment("__GODOT_EDITOR_CUSTOM_FEATURES__");
```

Comment:
Is there a reason the environment variable is surrounded by 2 underscores? I think the `EDITOR` makes it clear that it'll only work in editor builds.

For comparison, other environment variables used by Godot aren't surrounded by underscores.

Reply:
This is to signify that this variable is internal. Another example:
https://github.com/godotengine/godot/blob/da81ca62a5f6d615516929896caa0b6b09ceccfc/editor/editor_run.cpp#L306
Although maybe using it from command line makes sense.